src/index.js

```diff
@@ -73,19 +73,24 @@ class Radar {
       return;
     }
 
-    description = String(description).trim();
-    if (description.length === 0 || description.length > 256) {
+    const sanitizedDescription = String(description).trim();
+    if (sanitizedDescription.length === 0 || sanitizedDescription.length > 256) {
```

Comment:
I know this is legacy code, but I feel like we should drop the length checks here too. It seems like pretty unexpected behavior to just clear it out.